.gitignore

```diff
@@ -3,6 +3,12 @@
 .vscode
 **/.DS_Store
 
+# Data folders
+/supreme_court_predictions/data/convokit/*
+!/supreme_court_predictions/data/convokit/README.md
+/supreme_court_predictions/data/clean_convokit/*
+!/supreme_court_predictions/data/clean_convokit/README.md
```

Comment:
`.gitignore` things for the data folders.

Reply:
I don't think the data is proprietary, are we just doing this bc it's large?

Reply:
Yeah, it's just because it's a large amount of data.